### PR TITLE
Moved house rules pages

### PR DESCRIPTION
**Pull request by Unwrittend**
Reorganized house rules and created brewing crafting page

### CODE REVIEW

**Comment by Unwrittend** [comment]
Screenshot 2023-10-15 at 4 37 44 PM

**Comment by Unwrittend** [comment]
Includes minor updates to items updates page

File: house_rules/Combat Rules/resurrections.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Resurrection spells such as [Raise Dead](https://www.dndbeyond.com/spells/raise-
 - This penalty is reduced by 1 every time they complete a long rest.
 - Casting Revivify at a higher spell level reduces the penalty by 1 for every spell level higher than 3rd.
 
-There are rumors of a necromancer named [Rakkar](../npcs/rakkar) who can provide preventative measures against death for a fee. If you are wearing one of his soul crystals, you can be restored to life after death, back to a previously checkpointed level.
+There are rumors of a necromancer named [Rakkar](../../npcs/rakkar) who can provide preventative measures against death for a fee. If you are wearing one of his soul crystals, you can be restored to life after death, back to a previously checkpointed level.
 
 ![Resurrection light mode](/img/rules/resurrection.png#gh-light-mode-only)
```
Comment (Bcpoole) [inline house_rules/Combat Rules/resurrections.md]: Can you move the corresponding images into the jew directories as well?

**Review by Bcpoole — COMMENTED** [review]
Please either use a _ or - instead of a space in file paths.

Thanks for organizing!

File: house_rules/Combat Rules/resurrections.md (hunk 1 above)
Comment (Unwrittend) [reply house_rules/Combat Rules/resurrections.md]: Moved

**Pull request merged**